index.js

```diff
@@ -19,6 +19,7 @@
  * @return {void}
  */
 function newLine(state) {
+    if (!state.options.indentation && !state.options.lineSeparator) return;
```

Comment:
Although it might make the code "more" efficient, this line should not be needed.

Reply:
I'll accept the PR regardless.

Thanks

Reply:
> Although it might make the code "more" efficient, this line should not be needed.

@chrisbottin This line works if the option `{indentation: undefined, lineSeparator: undefined}` is given.